### PR TITLE
feature/team

### PR DESCRIPTION
**Pull request by hokhacminhtri**
Add team event.
Result:
- added_to_repository
image
- created
image
- deleted
image
- removed_from_repository
image

### CODE REVIEW

File: resources/views/events/github/team/added_to_repository.blade.php [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+?>
+
+⚡ <b>{!! __('tg-notifier::events/github/team.added_to_repository.title') !!}</b> 🎊
+
+@if(isset($payload->organization))
+🏢 {!! __('tg-notifier::events/github/team.added_to_repository.organization') !!}: <b>{{ $payload->organization->login }}</b>
+@endif
+@if(isset($payload->repository))
+📦 {!! __('tg-notifier::events/github/team.added_to_repository.full_name') !!}: 🦑<a href='{{ $payload->team->html_url }}'><b>{{ $payload->repository->full_name }}</b></a>
+🔐 {!! __('tg-notifier::events/github/team.added_to_repository.permissions') !!}: <b>{{ "{\n" . collect((array) $payload->repository->permissions)->filter()->map(fn($v, $k) => "    \"$k\": " . ($v ? '✔' : '❌'))->implode(",\n") . "\n}" }}</b>
```
Comment (tanhongit) [inline resources/views/events/github/team/added_to_repository.blade.php:16]: @hokhacminhtri what do you think about only displaying the highest permission, not the list?

Comment (hokhacminhtri) [reply resources/views/events/github/team/added_to_repository.blade.php:16]: @tanhongit Good suggestion, I'll update.

Comment (tanhongit) [reply resources/views/events/github/team/added_to_repository.blade.php:16]: thanks!

Comment (tanhongit) [reply resources/views/events/github/team/added_to_repository.blade.php:16]: @hokhacminhtri I created some tests, you just need to get the first permission name if it is true.

image image image

**Review by tanhongit — CHANGES_REQUESTED** [review]

File: resources/views/events/github/team/added_to_repository.blade.php (hunk 1 above)
Comment (tanhongit) [reply resources/views/events/github/team/added_to_repository.blade.php:16]: I checked the message and saw that we already showed the role, so I think there's no need to show the permission anymore, maybe it's not necessary. If you still want to display then there is no problem with this action anymore
image

File: resources/views/events/github/team/deleted.blade.php [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<?php
+
+/**
+ * @var $payload object
+ */
+
+?>
+
+⚡ <b>{!! __('tg-notifier::events/github/team.deleted.title') !!}</b> 🎊
```
Comment (tanhongit) [inline resources/views/events/github/team/deleted.blade.php:9]: Can you please change the trash icon for this line?

```suggestion
🗑 <b>{!! __('tg-notifier::events/github/team.deleted.title') !!}</b> 🎊
```

File: resources/views/events/github/team/added_to_repository.blade.php (hunk 1 above)
Comment (hokhacminhtri) [reply resources/views/events/github/team/added_to_repository.blade.php]: Updated. Please help me check it. Thanks.

**Review by tanhongit — APPROVED** [review]
thanks for all @hokhacminhtri

**Pull request merged**